### PR TITLE
Test noti

### PR DESCRIPTION
**Pull request by chipset508**

### CODE REVIEW

**Comment by chipset508** [comment]
aho

**Comment by chipset508** [comment]
hm

**Comment by chipset508** [comment]
hi

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def index
     if params[:search]
       @articles = Article.search(params[:search])
     else
+      puts 'Test noti'
```
Comment (chipset508) [inline app/controllers/articles_controller.rb]: h

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

File: app/controllers/articles_controller.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ def index
     if params[:search]
       @articles = Article.search(params[:search])
```
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 this is child comment 2

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: xin loi

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: work?

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: hm

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: hihi

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: what?

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: hm?

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: jjj

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: jj2

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: jj4.1

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: 4.2

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: 4.3

**Comment by chipset222** [comment]
nani

**Comment by chipset222** [comment]
wut

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: 4.4

**Comment by chipset222** [comment]
nani

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: 4.5

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: 4.6

**Comment by chipset222** [comment]
good

**Comment by chipset222** [comment]
nani

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: ngon

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: uicha

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: uicha2

**Comment by chipset222** [comment]
ngon

**Comment by chipset222** [comment]
choi

**Comment by chipset222** [comment]
hm

**Comment by chipset222** [comment]
iu

**Comment by chipset222** [comment]
na

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: ra1

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 mơ

**Comment by chipset222** [comment]
nhu e l

**Comment by chipset222** [comment]
hm

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 good

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 ahi

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 hu?

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: hoply

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: dc

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: ngon nha

**Comment by chipset222** [comment]
gut

**Comment by chipset222** [comment]
aha

**Comment by chipset222** [comment]
ahi

**Comment by chipset222** [comment]
nani

**Comment by chipset222** [comment]
hmm

**Comment by chipset222** [comment]
w

**Comment by chipset222** [comment]
deo

**Comment by chipset222** [comment]
?

**Comment by chipset222** [comment]
hm

**Comment by chipset222** [comment]
aha

**Comment by chipset222** [comment]
wut

**Comment by chipset222** [comment]
ahi

**Comment by chipset222** [comment]
aho

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 ngon day

**Comment by chipset222** [comment]
```
def abc
  a
end
```

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: aaaaaa

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: bbbb

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: ccccc

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: ddddd

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: e

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: f

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: g

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: h

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: x

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: u

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: q

**Comment by chipset508** [comment]
ok

**Comment by chipset222** [comment]
ha

**Comment by chipset222** [comment]
@chipset508

**Comment by chipset222** [comment]
@Tu Truong member

**Comment by chipset222** [comment]
@Tu Truong member

**Comment by chipset222** [comment]
@Tu Truong member  nani

**Comment by chipset222** [comment]
mention ne pa

**Comment by chipset222** [comment]
ha

**Comment by chipset222** [comment]
@chipset508 ahi

**Comment by chipset222** [comment]
@chipset508 nhìn nè, hay hok

`piu piu`

**Comment by chipset222** [comment]
úi, hay vồn @chipset508

**Comment by chipset222** [comment]
sao la @chipset508

**Comment by chipset222** [comment]
on r ne @chipset508

**Comment by chipset222** [comment]
GUT @chipset508

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 HAHAHAHAHA

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: KIDDING

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 kim @ahaha nani

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: nothing in your eyes

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 doi @ahia ahadfadsf asdf

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: some

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 dfsdfs adfasdfad sf a

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 ahiaiaiiaia

**Comment by chipset222** [comment]
```
TÂM SỰ TUỔI 30 - OST Phim Ông Ngoại Tuổi 30
Sáng tác: Trịnh Thăng Bình, Chí Tâm
Trình bày: Trịnh Thăng Bình
```

**Comment by chipset222** [comment]
Cawng

**Comment by chipset222** [comment]
@chipset508 nhac hay ko bay 
```
TÂM SỰ TUỔI 30 - OST Phim Ông Ngoại Tuổi 30
Sáng tác: Trịnh Thăng Bình, Chí Tâm
Trình bày: Trịnh Thăng Bình
```

**Comment by chipset222** [comment]
@chipset508 
`nhieeuf lan muon dung lai nghi ngoiiii`

**Comment by chipset222** [comment]
K quote dc r

**Comment by chipset222** [comment]
em chua muoi tam

**Comment by chipset222** [comment]
`hihi`

**Comment by chipset222** [comment]
```
def
  ahi
  hihi
end
```

**Comment by chipset222** [comment]
<code> hiih </code>

**Comment by chipset222** [comment]
hm

**Comment by chipset222** [comment]
go

**Comment by chipset222** [comment]
len

**Comment by chipset222** [comment]
choi

**Comment by chipset222** [comment]
ngon

**Comment by chipset222** [comment]
cup

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
a

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: a

**Comment by chipset222** [comment]
len

**Comment by chipset222** [comment]
ngon

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 aaaa

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
a @chipset508

File: app/controllers/articles_controller.rb (hunk 2 above)
Comment (chipset222) [inline app/controllers/articles_controller.rb]: hm @chipset508

**Comment by chipset222** [comment]
aha

**Comment by chipset222** [comment]
a

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 adsfasdfasdf ádf ádf ádf ádfasdf adfaf adsf ádf dfdasf sd
```fdfdfasdfdasfdsfsaf````
đfdsfsdf dsfg sdfgdfg sdfg

[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 ádfadfadfas dfadfa sdf à

**Comment by chipset222** [comment]
@xasfjioasdfjsd afadf

**Comment by chipset222** [comment]
aaaaaa @chipset508

**Comment by chipset222** [comment]
@chipset508  gut

File: app/controllers/articles_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipset222) [reply app/controllers/articles_controller.rb]: @chipset508 aaaafadf afadf adfafa d

**Comment by chipset222** [comment]
ha

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
af

**Comment by chipset222** [comment]
af

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
fad

**Comment by chipset222** [comment]
fa

**Comment by chipset222** [comment]
fadf

**Comment by chipset222** [comment]
aa

**Comment by chipset222** [comment]
aha

**Comment by chipset222** [comment]
afa

**Comment by chipset222** [comment]
af

**Comment by chipset222** [comment]
ff

**Comment by chipset222** [comment]
af

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
fa

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
faf

**Comment by chipset222** [comment]
afaf

**Comment by chipset222** [comment]
aha

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
b

**Comment by chipset222** [comment]
c

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
d

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
gut

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
d

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
c

**Comment by chipset222** [comment]
dd

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
sa

**Comment by chipset222** [comment]
s

**Comment by chipset222** [comment]
@chipset508

**Comment by chipset222** [comment]
@chipset508  aaa

**Comment by chipset222** [comment]
@chipset508 a

**Comment by chipset222** [comment]
hi

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
b

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
f

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
b

**Comment by chipset222** [comment]
c

**Comment by chipset222** [comment]
v

**Comment by chipset222** [comment]
d

**Comment by chipset222** [comment]
a

**Comment by chipset222** [comment]
c

**Comment by chipset222** [comment]
af

**Comment by chipset222** [comment]
v

**Comment by chipset222** [comment]
x

**Pull request merged**